tests/e2e/tests/content-manager/history.spec.ts

```diff
@@ -1,18 +1,25 @@
 import { test, expect } from '@playwright/test';
 import { login } from '../../utils/login';
-import { resetDatabaseAndImportDataFromPath } from '../../scripts/dts-import';
+import { resetDatabaseAndImportDataFromPath } from '../../utils/dts-import';
```

Comment:
Looking at the CI it looks like `e2e/tests/admin/api-tokens.spec.ts` needs that fix too, could you add it there as well?

Reply:
Good catch, yeah I'll do that now, thanks